styles/_globals.scss

```diff
@@ -1,2 +1,47 @@
+html {
+  font-size: 16px;
+}
 
+body {
+  background-color: $body-bg-color;
+}
 
+.container {
+  margin-right: auto;
+  margin-left: auto;
+  padding-left: $pd-mobile;
+  padding-right: $pd-mobile;
+  max-width: 21.438rem;
+
+  @include tablet {
+    padding-left: $pd-tablet;
+    padding-right: $pd-tablet;
+    max-width: 46rem;
+  }
+
+  @include desktop {
+    padding-left: $pd-desktop;
+    padding-right: $pd-desktop;
+    max-width: 78.375rem;
+  }
+}
+
+.manrope-500 {
```

Comment:
I would prefer calling it something like "primary-font", without referring directly to the font family name